editor/editor_help.cpp

```diff
@@ -2494,11 +2494,34 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 			} else if (link_tag == "annotation") {
 				target_color = link_annotation_color;
 			}
+
 			p_rt->push_color(target_color);
 			p_rt->push_meta("@" + link_tag + " " + link_target);
-			p_rt->add_text(link_target + (link_tag == "method" ? "()" : ""));
-			p_rt->pop();
-			p_rt->pop();
+
+			if (link_tag == "member" &&
+					((!link_target.contains(".") && (p_class == "ProjectSettings" || p_class == "EditorSettings")) ||
+							link_target.begins_with("ProjectSettings.") || link_target.begins_with("EditorSettings."))) {
+				// Special formatting for both ProjectSettings and EditorSettings.
+				String prefix;
+				if (link_target.begins_with("EditorSettings.")) {
+					prefix = "(Editor) ";
```

Comment:
This should be localized.

Also not showing anything for Project Settings makes me wonder if it's Project Settings or Editor Settings (but not a problem once you know the rules).

Reply:
I thought it was okay, since Project Settings are MUCH more common than Editor Settings. When clicking on the setting the question is answered, too.
I was considering more settings that visualize like this (**EditorExportPlatform** & co.), and then it would actually be problem. But that remains to be seen...